core/components/menu/menuitem.js

```diff
@@ -232,51 +217,39 @@ Blockly.MenuItem.prototype.setEnabled = function(enabled) {
 
   var el = this.getElement();
   if (el) {
-    if (!this.enabled_) {
-      Blockly.utils.dom.addClass(el, 'goog-menuitem-disabled');
+    // goog-menuitem-disabled is deprecated, use blocklyMenuItemDisabled.
```

Comment:
A link back to the comment above, this is what I'm referring to.https://github.com/google/blockly/pull/3880/files#r421074552